R/html-tweak.R

```diff
@@ -163,6 +154,33 @@ tweak_homepage_html <- function(html, strip_header = FALSE) {
   invisible()
 }
 
+tweak_sidebar_html <- function(html, sidebar) {
+  if (!sidebar) {
+    return(html)
+  }
+  # Only add badges if there is a sidebar,
```

Comment:
Now you can remove this comment since it's clear what's happening from the code 😄